packages/loader/container-loader/src/container.ts

```diff
@@ -1619,11 +1619,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             if (errorMsg !== undefined) {
                 const error = new GenericError(
                     errorMsg,
+                    undefined,
                     {
                         clientId: this._clientId,
                         messageClientId: message.clientId,
                         sequenceNumber: message.sequenceNumber,
                         clientSequenceNumber: message.clientSequenceNumber,
+                        messageTimestamp: message.timestamp,
```

Comment:
I'm fine with this, but i think we might actually want to just add a bunch of this information directly to the logger at the container level, so these field are always avaialbe, but maybe it's overkill. or maybe only add them for errors

Reply:
+1 to that. I feel like we should always log certain things like clientId and timestamp.

Reply:
#5121 

Good idea.

Reply:
Note: As a general rule, we try not to add info that already exists in telemetry. clientId (not message clientID, but current known to container) is already in the logs. There is pros & cons to either approach (each event is self-contained vs. not), but I spent some time analyzing cost of logging (Kusto has reports for that) and things like clientID are unfortunately rather expensive when repeated despite compression (i.e. that's the reason I removed it from being on all events).
In this case it's likely not a big deal (error handling - rare events), so mostly FYI for future.